lib/packwerk/parsers/erb.rb

```diff
@@ -35,12 +40,18 @@ def parse_buffer(buffer, file_path:)
 
       private
 
+      sig do
+        type_parameters(:T).params(
+          erb_ast: T.all(T.type_parameter(:T), Object),
+          file_path: String
+        ).returns(T.type_parameter(:T))
+      end
       def to_ruby_ast(erb_ast, file_path)
```

Comment:
Can the `erb_ast` be many things? Could/should we just enumerate what it can be? Why generic?

Reply:
So it turns out the AST value can either be a node, a string, and nil. Using these types in a `T.any` clause works.